pom.xml

```diff
@@ -255,6 +274,11 @@
                 <version>2.9</version>
             </plugin>
             <plugin>
+                <groupId>org.jacoco</groupId>
```

Comment:
NM - that's for the integration test execution.

Try `mvn clean package site:site`.